hack/trivy_scan.py

```diff
@@ -54,7 +55,7 @@ def save_images(wg, images, version):
     output_file = f"../image_lists/kf_{version}_{wg}_images.txt"
     with open(output_file, 'w') as f:
         f.write('\n'.join(images))
-    print(f"File {output_file} successfully created")
+    print(f"File {output_file} successfully created", flush=True)
```

Comment:
Wouldn't it make sense to create a `log` function (or name something else) which is just `print(..., flush=True)`?

Reply:
I made the change